packages/sdk/src/tests/assets/open-edition-nft.test.ts

```diff
@@ -48,35 +48,18 @@ describe('Asset class: OpenEditionNFT', () => {
   beforeEach(() => {
     nft = new OpenEditionNFT({
       contract,
-      metadata: {
-        name: 'NFTs Without Boundaries',
-        description: 'Probably nothing.',
-        creator: 'Holograph Protocol',
-        attributes: {
-          key: 'value',
-          anotherKey: 'anotherValue',
-        },
-      },
     })
   })
 
-  it('should be able to get the OpenEditionNFT wrapper class', () => {
+  it.skip('should be able to get the OpenEditionNFT wrapper class', () => {
     expect(nft).toHaveProperty('metadata')
     expect(nft).toHaveProperty('name')
     expect(nft).toHaveProperty('description')
-    expect(nft).toHaveProperty('creator')
-    expect(nft).toHaveProperty('attributes')
-    expect(nft).toHaveProperty('ipfsUrl')
+    expect(nft).toHaveProperty('properties')
     expect(nft).toHaveProperty('ipfsImageCid')
+    expect(nft).toHaveProperty('animationUrl')
     expect(nft).toHaveProperty('getOwner')
     expect(nft).toHaveProperty('getParsedTokenId')
-    expect(nft).toHaveProperty('setMetadata')
```

Comment:
we need these kind of methods back i guess

Reply:
We can't set the metadata for an OE NFT, it's defined when we deploy the OE collection. The NFT metadata is then derived from the collection's metadata.